### ISSUE TITLE
Interrupted scans: follow ReproNim recommendations

### ISSUE THREAD

**Issue by celprov** [TheAxonLab/hcph-sops#190]
Incorporate ReproNim recommendations on how to deal with interrupted scans into the SOPs

https://github.com/ReproNim/reproin/blob/master/docs/walkthrough-1.md#interrupted-program

**Comment by celprov** [comment]
I marked this item as urgent, because we need to make sure to follow the ReproNim recommendations in case a scan has to be interrupted

**Comment by celprov** [comment]
@helenelajous @esavary @oesteban @acionca sorry, I misled us this morning. We were looking at the wrong section of the ReproNim recommendations.
They differentiate between two scenarios where you have to repeat the scan:
1. there is a problem in the scan, so you have to [restart that particular scan](https://github.com/ReproNim/reproin/blob/master/docs/walkthrough-1.md#interrupted-scan): in that case you should repeat **without changing sequence name**
2. Adding the `_run` is recommended in case you have to [repeat some sequences because you had to get the subject out of the scanner](https://github.com/ReproNim/reproin/blob/master/docs/walkthrough-1.md#interrupted-program) for some problem and you're putting him back in. You have to rerun e.g the T1w, but you need to keep both the T1w before and after participant's exit. But I think Oscar's code is already detecting and assigning `_run` in the BIDS name to save both scans, so it is not important for us.

In summary, in any case, you should use the `Repeat` button and **not** change the name of the sequence
